test-utils/setupTestFramework.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+const consoleError = console.error
+
+beforeEach(() => {
+  // Suppress errors from JSDOM CSS parser
+  // See: https://github.com/jsdom/jsdom/issues/2177
+  // eslint-disable-next-line flowtype-errors/show-errors
+  ;(console: any).error = message => {
+    if (!message.includes('Error: Could not parse CSS stylesheet')) {
+      consoleError(message)
+    }
+  }
+})
+
+afterEach(() => {
+  // eslint-disable-next-line flowtype-errors/show-errors
+  ;(console: any).error = consoleError
```

Comment:
`;(console: any).error` what is this magic?

Reply:
Ask Prettier :D

Reply:
ha! It'd make sense if the previous line required a semicolon to be put there but better safe than sorry ?! 😆